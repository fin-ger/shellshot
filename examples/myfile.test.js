const { setup } = require('shellshot');

setup();

it(
    'checks if the content of the file myfile is still the same',
    async () => {
        await expect.file('./res/myfile').toMatchSnapshot();
    },
);
