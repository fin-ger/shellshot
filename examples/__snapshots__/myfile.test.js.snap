// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks if the content of the file myfile is still the same 1`] = `
Object {
  "data": Array [
    72,
    101,
    108,
    108,
    111,
    44,
    32,
    119,
    111,
    114,
    108,
    100,
    33,
    10,
  ],
  "type": "Buffer",
}
`;
